### ISSUE TITLE
dependency versions are missing from pom

### ISSUE THREAD

**Issue by tglanz** [rapiddweller/rapiddweller-benerator-ce#45]
Hi, am not sure if this is a bug or misuse by me.

**Describe the bug**
I use ```bin/benerator-wizard``` to generate a ```Simple project```. The project is generated successfully but the dependency versions are missing from the ```pom.xml``` file.

**To Reproduce**
1. run ```bin/benerator-wizard```
2. set ```Project Folder``` and ```Project Name```
3. click ```Create```
4. open file ```pom.xml``` in the generated folder and see that there are not properties defined for 
  * ```version.dependencies.dbsanity```
  * ```version.dependencies.rd-lib-jdbacl```
  * ```version.dependencies.commons```

**Expected behavior**
The ```pom.xml``` file to contain a properties section with values specified for the above

**Screenshots**
Below is the generated ```pom.xml``` file

```xml
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">

    <modelVersion>4.0.0</modelVersion>

    <groupId>com.my</groupId>
    <artifactId>first</artifactId>
    <version>1.0</version>

    <packaging>jar</packaging>
    <name>first data generation project</name>

    <dependencies>
        <dependency>
            <groupId>com.rapiddweller</groupId>
            <artifactId>rapiddweller-benerator-ce</artifactId>
            <version>1.1.2-jdk-11</version>
        </dependency>
        <dependency>
            <groupId>org.databene</groupId>
            <artifactId>dbsanity</artifactId>
            <version>${version.dependencies.dbsanity}</version>
        </dependency>
        <dependency>
            <groupId>com.rapiddweller</groupId>
            <artifactId>rd-lib-jdbacl</artifactId>
            <version>${version.dependencies.rd-lib-jdbacl}</version>
        </dependency>
        <dependency>
            <groupId>com.rapiddweller</groupId>
            <artifactId>rd-lib-format</artifactId>
            <version>1.1.2-jdk-11</version>
        </dependency>
        <dependency>
            <groupId>com.rapiddweller</groupId>
            <artifactId>rd-lib-common</artifactId>
            <version>${version.dependencies.commons}</version>
        </dependency>
        <dependency>
            <groupId>com.rapiddweller</groupId>
            <artifactId>rd-lib-contiperf</artifactId>
            <version>2.4.0-jdk-11</version>
        </dependency>
    </dependencies>

    <build>

        <plugins>

            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <configuration>
                    <encoding>UTF-8</encoding>
                    <source>11</source>
                    <target>11</target>
                </configuration>
            </plugin>

            <plugin>
                <groupId>org.databene</groupId>
                <artifactId>maven-benerator-plugin</artifactId>
                <version>1.1.2-jdk-11</version>

                <configuration>
                    <descriptor>benerator.xml</descriptor>
                    <encoding>UTF-8</encoding>
                    <validate>true</validate>
                </configuration>

                <dependencies>
        <!-- logging -->
        <dependency>
            <groupId>org.apache.logging.log4j</groupId>
            <artifactId>log4j-api</artifactId>
            <version>2.14.0</version>
            <scope>provided</scope>
        </dependency>
        <dependency>
            <groupId>org.apache.logging.log4j</groupId>
            <artifactId>log4j-core</artifactId>
            <version>2.14.0</version>
            <scope>provided</scope>
        </dependency>
                </dependencies>
            </plugin>

        </plugins>
    </build>

</project>
```

**Desktop (please complete the following information):**
 - OS: Arch Linux (kernel: 5.12.8-arch1-1)

**Additional context**
I used release [1.1.2-jdk-11](https://github.com/rapiddweller/rapiddweller-benerator-ce/releases/tag/1.1.2)

Thank you for this great product!

**Comment by ake2l** [comment]
Hi thanks for your detailed issue !
Actually the wizard was not so much in scope when it comes to refactoring. I adjusted the related archetypes with new dependencies, but spent not so much time with testing these, so this is probably a bug.

I am going to check and fix it.

**Comment by ake2l** [comment]
here we come with a quickfix :)
this is how the pom should look like, than you can use `mvn benerator:generate` to generate your data

`<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">

    <modelVersion>4.0.0</modelVersion>

    <groupId>com.my</groupId>
    <artifactId>simple</artifactId>
    <version>1.0</version>

    <packaging>jar</packaging>
    <name>simple data generation project</name>

    <dependencies>
        <dependency>
            <groupId>com.rapiddweller</groupId>
            <artifactId>rapiddweller-benerator-ce</artifactId>
            <version>1.1.2-jdk-11</version>
        </dependency>
    </dependencies>

    <build>

        <plugins>

            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <configuration>
                    <encoding>UTF-8</encoding>
                    <source>11</source>
                    <target>11</target>
                </configuration>
            </plugin>

            <plugin>
                <groupId>org.databene</groupId>
                <artifactId>maven-benerator-plugin</artifactId>
                <version>0.7.7</version>

                <configuration>
                    <descriptor>benerator.xml</descriptor>
                    <encoding>UTF-8</encoding>
                    <validate>true</validate>
                </configuration>

                <dependencies>
                    
                    <dependency>
                        <groupId>org.apache.logging.log4j</groupId>
                        <artifactId>log4j-api</artifactId>
                        <version>2.14.0</version>
                    </dependency>
                    <dependency>
                        <groupId>org.apache.logging.log4j</groupId>
                        <artifactId>log4j-core</artifactId>
                        <version>2.14.0</version>
                    </dependency>
                </dependencies>
            </plugin>

        </plugins>
    </build>

</project>
`

**Comment by ake2l** [comment]
This will be fixed with #49

**Comment by tglanz** [comment]
Hi, thanks for your feedback and for the fix!

Note - I had to remove the "configuration" section actually. otherwise I got a "No plugin found for prefix 'benerator'".

**Comment by ake2l** [comment]
Hi , you are welcome but this topic is still ongoing, we recreated the original maven plugin [here](https://github.com/rapiddweller/benerator-maven-plugin)  , you might have seen .
This one is using the new rapiddweller version > 1.x of the benerator. 
The rapiddweller-benerator-ce 1.2.0 wizard will use this new plugin out of the box.

At the moment you can use the new plugin like this 

```xml
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">

    <modelVersion>4.0.0</modelVersion>

    <groupId>com.my</groupId>
    <artifactId>myproject</artifactId>
    <version>1.0</version>

    <packaging>jar</packaging>
    <name>myproject data generation project</name>
    <description>Start CSV file generation by typing 'mvn benerator:generate' on the command line.</description>



    <build>

        <plugins>

            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <configuration>
                    <encoding>UTF-8</encoding>
                    <source>11</source>
                    <target>11</target>
                </configuration>
            </plugin>

            <plugin>
                <groupId>com.rapiddweller</groupId>
                <artifactId>benerator-maven-plugin</artifactId>
                <version>1.1.2-jdk-11</version>

                <configuration>
                    <descriptor>benerator.xml</descriptor>
                    <encoding>UTF-8</encoding>
                    <validate>true</validate>
                </configuration>

                <dependencies>
                    <!-- logging -->
                    <dependency>
                        <groupId>org.apache.logging.log4j</groupId>
                        <artifactId>log4j-api</artifactId>
                        <version>2.14.0</version>
                    </dependency>
                    <dependency>
                        <groupId>org.apache.logging.log4j</groupId>
                        <artifactId>log4j-core</artifactId>
                        <version>2.14.0</version>
                    </dependency>
                    <!-- graalvm-->
                    <dependency>
                        <groupId>org.graalvm.js</groupId>
                        <artifactId>js</artifactId>
                        <version>21.1.0</version>
                    </dependency>
            
                    <dependency>
                        <groupId>org.graalvm.sdk</groupId>
                        <artifactId>graal-sdk</artifactId>
                        <version>21.1.0</version>
                    </dependency>
            
                </dependencies>

            </plugin>

        </plugins>
    </build>

</project>
```

the graalvm part as dependency won't be necessary for the next version, because it will come from the benerator lib.
If you run a benerator script with the new plugin, at the end of your console Log you will find 

```java
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:39 min
[INFO] Finished at: 2021-07-12T08:45:08+02:00
[INFO] ------------------------------------------------------------------------
Exception in thread "Thread-3" java.lang.IllegalAccessError: failed to access class org.graalvm.collections.EconomicMapImpl$3 from class org.graalvm.collections.EconomicMapImpl (org.graalvm.collections.EconomicMapImpl$3 is in module org.graalvm.sdk of loader 'app'; org.graalvm.collections.EconomicMapImpl is in unnamed module of loader org.codehaus.plexus.classworlds.realm.ClassRealm @189b5fb1)
	at org.graalvm.collections.EconomicMapImpl.iterator(EconomicMapImpl.java:846)
	at com.oracle.truffle.polyglot.PolyglotEngineImpl.collectAliveContexts(PolyglotEngineImpl.java:1177)
	at com.oracle.truffle.polyglot.PolyglotEngineImpl.ensureClosed(PolyglotEngineImpl.java:1067)
	at com.oracle.truffle.polyglot.PolyglotEngineImpl$PolyglotShutDownHook.run(PolyglotEngineImpl.java:1396)
	at java.base/java.lang.Thread.run(Thread.java:829)

```

this has no impact on your beneration process!
This is an issue with a shutdown hook implemented in graalvm (PolyglotShutDownHook), we also have an issue for this in benerator maven plugin project ( [here](https://github.com/rapiddweller/benerator-maven-plugin/issues/1) )

**Comment by ake2l** [comment]
fixed with #55
